napari_plugin_manager/styles.qss

```diff
@@ -350,3 +350,26 @@ QtLayerList::indicator:checked {
   min-height: 36px;
   min-width: 36px;
 }
+
+QToolButton {
+  background-color: {{ darken(foreground, 20) }};
+  color: {{ darken(text, 15) }};
+  padding: 3 12px;
+  font: 14px;
+}
+
+QToolButton:hover {
+  background-color: {{ lighten(foreground, 10) }}
+}
+
+QToolButton:pressed {
+  background-color: {{ lighten(foreground, 20) }}
+}
+
+QToolButton::menu-arrow {
+  image: url("theme_{{ id }}:/down_arrow.svg");
+  top: 0px;
+  left: 0px;
+  width: 5px;
+  height: 5px;
+}
```

Comment:
With this style changes (width and heigth set to 5px) on Windows I'm seeing something like this:

![imagen](https://github.com/user-attachments/assets/ca0ade27-069e-466e-96b1-6e0820dfefaa)

Also, for reference, on Windows:

* If I change the width and height to something like 8px I'm able to see something like this:

![imagen](https://github.com/user-attachments/assets/3a84d04f-ce29-40e7-8d22-e2fabfb35abf)

And without any custom `menu-arrow` style:

![imagen](https://github.com/user-attachments/assets/840a0f57-55ca-4917-8b91-5c0ec3735f0f)

Reply:
Thanks for the input, however I could not make it work on Mac with bugger values, and that is why I left it "small" 5px

Reply:
@dalthviz this is how it looks on mac, it seems I do not have control over the actual size of the icon nor the spacing to the sides of it :-|  on mac at least.

<img width="1012" alt="Screenshot 2024-08-18 at 8 32 53 PM" src="https://github.com/user-attachments/assets/3b1aab47-2b28-4ab5-ad5a-e5a4ccf0f1be">

Reply:
It seems on mac I cannot control the width of the  `MenuButtonPopup` thinguie

```
QToolButton::menu-arrow {
  image: url("theme_{{ id }}:/down_arrow.svg");
  top: 0px;
  left: 0px;
  width: 10px;
  height: 10px;
}
```

The arrow increases in size, but not the rectangular  section

<img width="106" alt="Screenshot 2024-08-18 at 8 41 41 PM" src="https://github.com/user-attachments/assets/806a6ca7-542d-4659-9b27-b5e7a7dae4d3">

Reply:
Oh I see, indeed the right clickable path is quite different on macOS than on Windows 🤔 will try to give a check on macOS to see if there is something to style that part :+1:

Reply:
Seems like what is needed is to use the `QToolButton[popupMode="1"]` to give the overall button extra width (only when the pop up mode `MenuButtonPopup`) and for the arrow button itself you can use to style things `QToolButton::menu-button`

For more info you can check: https://doc.qt.io/qt-5/stylesheet-examples.html#customizing-qtoolbutton

Reply:
Setting as style rules something like:

```
QToolButton::menu-arrow {
  image: url("theme_{{ id }}:/down_arrow.svg");
  top: 0px;
  left: 0px;
  width: 8px;
  height: 8px;
}

QToolButton[popupMode="1"]{
padding-right: 20px;
}

QToolButton::menu-button {
 width: 16px;
}

```

I was able to get on macOS something like:

<img width="138" alt="image" src="https://github.com/user-attachments/assets/522e3dd2-e630-43b6-8646-5b7296da9cd3">

Reply:
Awesomeeee @dalthviz indeed I did not try that!

Thanks, will implement the changes and we are good to go :)

Reply:
![Uploading Screenshot 2024-08-19 at 12.35.27 PM.png…]()